### ISSUE TITLE
the result of about coco person are not good

### ISSUE THREAD

**Issue by cendelian** [zylo117/Yet-Another-EfficientDet-Pytorch#245]
the result of about coco person are not good!

Hi, I train my person data(come from coco). and the train command :
python person_train.py --load_weights ./weights/efficientdet-d0.pth

after 5 epoch, I test the jpg, the result is
**Val. Epoch: 5/100. Classification loss: 0.14459. Regression loss: 0.02242. Total loss: 0.16701
Step: 28861. Epoch: 6/100. Iteration: 14/4808. Cls loss: 0.09186. Reg loss: 0.01447. Total loss: 0.10633:   0%|          | 14/4808 [00:06<38:31,  2.07it/s]**

and I use the efficientdet-d0_5_27000.pth the test jpg, the result is not good as below:

![image](https://user-images.githubusercontent.com/50360389/80865513-8e751700-8cbc-11ea-967d-0934297c8f14.png)

why the result is so bad? the number of epoch is not enough?
could you give me some advice?
By the way, the train learning rate is 1e-4.

Thanks!!

**Comment by singlautsav** [comment]
How many classes are you training.
Just on the side note one needs to have lower loss. So in case you have less number of classes the loss must be of the order 5*10^-4. and 5 epoch are really nothing you have to let it work for sometime.

**Comment by zylo117** [comment]
This image is difficult to detect. You should try running on your val set.

**Comment by merisgong208** [comment]
I tried efficientdet-d7.pth, why I cannot get that many objects described in Readme.md? @zylo117

**Comment by zylo117** [comment]
@merisgong208 set `force_input_size=1920` if you are using coco pretrained weights

**Comment by cendelian** [comment]
> How many classes are you training.
> Just on the side note one needs to have lower loss. So in case you have less number of classes the loss must be of the order 5*10^-4. and 5 epoch are really nothing you have to let it work for sometime.

just one class(person).
I think my learning rate is big for pretrain.
and I try 1e-5 now and train more epochs

**Comment by cendelian** [comment]
> This image is difficult to detect. You should try running on your val set.

thanks.
because my data come from coco, and just one class(person).
So i use this img to test.

**Comment by zylo117** [comment]
I don't know if this image comes from coco. I just took it from the official impl. You should test it on val set.
